opi/bob/evg/evm-mtca-300.bob

```diff
@@ -1162,10 +1133,6 @@ $(pv_value)</tooltip>
       <color name="Header_2_ForeGround" red="255" green="255" blue="255">
       </color>
     </foreground_color>
-    <line_color>
-      <color name="Header_2_Background" red="114" green="159" blue="207">
```

Comment:
You've removed all the "Header_2_Background" please put it back. 
Otherwise It uses the Macro "Grid" by default.
You can use 
Header_2_Background=Grid in your color.def if you prefer the original Grid color.

If you can be compliant with that in your VME bobs, it would be perfect too.

Reply:
I can add back the foreground_color section:
`<foreground_color>
      <color name="Header_2_ForeGround" red="255" green="255" blue="255">
      </color>
    </foreground_color>
`
The line_color section causes an error while loading the screen.
`<line_color>
      <color name="Header_2_Background" red="114" green="159" blue="207">
      </color>
    </line_color>
`
![image](https://github.com/user-attachments/assets/5c2a23f8-5f52-4b02-a629-6174215fb74b)
After saving the screen I found that cssphoebus removed this section. Seems that the group widget does not have a property line_color.

I just updated the screens so all groups have the Header_2_ForeGround as the foreground_color.

Reply:
Line_color was added by a student of us few month ago, I think you are not using the last version of Phoebus. I am using Phoebus v4.7.4.
I should indicate the minimum version of the bob in the README